app/models/episode_story_handler.rb

```diff
@@ -60,6 +60,11 @@ def update_attributes
     episode.content = sa[:description]
     episode.categories = sa[:tags]
     episode.published_at = sa[:published_at] ? Time.parse(sa[:published_at]) : nil
+
+    season_num = sa[:season_identifier].to_i
+    episode.season_number = season_num unless season_num.blank? || season_num.zero?
```

Comment:
might someone have a season 0? or an episode 0 for a preview episode? I think you need to check for blank  value before you call to_i